### ISSUE TITLE
Feature request - Post Type Single Mapping

### ISSUE THREAD

**Issue by danaskallman** [ronalfy/post-type-archive-mapping#44]
The options for Archive mapping in the Reading Settings, plus the term/taxonomy selection is amazing. Could this be done for the single template too? Meaning a page could be setup for a Single Post Item using the blocks available to display 1 item and an option to display full content? This would be particularly helpful with the ability to use custom fields that's availablle. 

Again thanks for this very useful plugin!

**Comment by ronalfy** [comment]
I'm afraid I don't follow. Do you happen to have an example I can look at?

Thanks,

Ronald

PS: If you haven't already, can you please leave us a rating? We'd really appreciate it. https://wordpress.org/support/plugin/post-type-archive-mapping/reviews/

**Comment by danaskallman** [comment]
Of course, I thought it may be confusing. I setup a quick example. I am able to create a `Member` CPT with some custom fields which you can see here: https://glocal.works/members/, I've set that as the archive in the Reading setting. 

But when you click on the View Profile to see more it takes you to the default theme single: https://glocal.works/members/saturn-the-destroyer/, but it would be great to use the block and setup a single plate like this: https://glocal.works/member-profile/ settings up would looke something like this:

![member-single-page-template-example](https://user-images.githubusercontent.com/1139085/80507188-904d7c00-8944-11ea-951d-59426d4c6548.png)

Would it be possible to have something like this:

![member-single-page-template-single](https://user-images.githubusercontent.com/1139085/80508003-7b251d00-8945-11ea-8610-240ea808256a.png)

So it could redirect to that page for the CPT single as opposed to default?

**Comment by ronalfy** [comment]
That sounds reasonable. I think in order to do this, I'll have to provide some type of layout builder in the plugin so you can create a block, assign a layout, and another custom post types block that is more configurable than the current solution. It's slated for 5.0.0, but since I'm the only one working on this currently, I can't speculate when it'll be released.
